scthumbd.js

```diff
@@ -57,18 +61,39 @@ const thumber = scThumber({
 const workers = process.env.WORKERS ?? os.cpus().length;
 const port = process.env.PORT ?? 4001;
 
-if (cluster.isPrimary && workers > 1) {
-  console.log(`${'[m]'.red} ${'scthumbd %s'.yellow}`, process.env.npm_package_version);
+if (cluster.isPrimary) {
+  const { version } = JSON.parse(fs.readFileSync('./package.json').toString());
```

Comment:
this doesn't work if the process is started anywhere not the app root directory. That said, I don't know if the version display is useful at all especially since it's never updated. Might as well just remove it.

```
$ node scthumber/scthumbd.js
node:internal/fs/sync:78
  return binding.openSync(
                 ^

Error: ENOENT: no such file or directory, open './package.json'
    at Object.open (node:internal/fs/sync:78:18)
    at Object.openSync (node:fs:565:17)
    at Object.readFileSync (node:fs:445:35)
    at file:///home/edho/osu/scthumber/scthumbd.js:65:37
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: './package.json'
}

Node.js v20.8.0
```

Reply:
That was in my mind but thought we just wouldn't support launching it from anywhere but the root. That said, I also agree with removing the version display